src/project-builder.ts

```diff
@@ -2,55 +2,44 @@ import { spawnSync } from "node:child_process";
 import { writeFileSync } from "node:fs";
 import { chalk } from "./chalk.js";
 import prompt from "enquirer";
-import sentencer from "sentencer";
 import exampleConfig from "./tableland.config.example.js";
 
 const docsLink = "https://docs.tableland.xyz";
 const githubLink = "https://github.com/tablelandnetwork";
 
 export const projectBuilder = async function () {
-  const choices = [
-    "No, I don't want to create a project right now",
-    "Yes, and I want to control everything each step of the way",
-    "Yes, but setup everything for me and stop asking me questions",
-  ];
+  const choices = ["No", "Yes"];
   // @ts-ignore https://github.com/enquirer/enquirer/issues/379
   const select = new prompt.Select({
     name: "wtd",
-    message:
-      "Couldn't find an existing Tableland project, do you want to create a new project?",
+    message: "Welcome to tableland, do you want to create a new project?",
```

Comment:
slick. is this meant to function like hardhat projects? as in, an easy way to create a local tableland config file?

Reply:
That's the idea, but I've been wondering how useful that will be if only tableland core team will need that kind of setup?

Reply:
for apps like example-voter, it sounds useful right?

Reply:
Depends how the app want to use this.
For the example-voter app I was thinking for tests you could (after this pr) do `npm install --save-dev @tableland/local` and be ready to use this in your tests via `import { LocalTableland } from "@tableland/local";`
If you want to run an app with a client against this, you would do `npx local-tableland` and be good to go.

The only use-cases we talked about that needs a tableland.config.js file is the core development.  That could change as we add more features though.

Reply:
oh, i see... makes sense. in that case, i think you can ignore my comment about using a flag below if this project builder is meant to only be used with docker against a local go-tableland repo.